rust/stackable-cockpit/src/engine/kind/mod.rs

```diff
@@ -75,6 +75,8 @@ impl KindCluster {
             .args(["create", "cluster"])
             .args(["--name", self.name.as_str()])
             .args(["--config", "-"])
+            .stdout(Stdio::null())
+            .stderr(Stdio::null())
```

Comment:
Can we please.. not silence helpful logs of subprocesses?

Reply:
This will interfere with the spinner. I want to address this in an upcoming PR.

Reply:
Then let's not do the spinner (plain text logs instead?) until we can do that.

Reply:
Agreed. Let's add the spinner in a separate PR.

Reply:
Removed in 558affd